### PR TITLE
Added a guess the item game example

### PR DESCRIPTION
**Pull request by rosswf**

### CODE REVIEW

File: osrsbox/items_api_examples/guess_the_item_game.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+from osrsbox import items_api
+import random
+
+if __name__ == "__main__":
+    all_db_items = items_api.load()
+    item_found = False
+    item_name = ""
+    while item_found == False:
+        random_id = random.randint(1,len(all_db_items))
```
Comment (osrsbox) [inline osrsbox/items_api_examples/guess_the_item_game.py]: Flake8 is complaining about this one too. There is missing white space after the comma (`,`). 

`random_id = random.randint(1, len(all_db_items))`

File: osrsbox/items_api_examples/guess_the_item_game.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        for item in all_db_items:
+            # Discard items with examine text on None, this is not a required field.
+            if item.examine != None and item.id == random_id:
```
Comment (osrsbox) [inline osrsbox/items_api_examples/guess_the_item_game.py]: Flake8 is complaining about this one too. 

`if item.examine != None`

Should be:

`if item.examine is not None`

File: osrsbox/items_api_examples/guess_the_item_game.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                print(item.examine)
+                item_found = True
+                item_name = item.name
+
+    answer = input("What is this item? ")
+
+    if answer == item_name:
```
Comment (osrsbox) [inline osrsbox/items_api_examples/guess_the_item_game.py]: This is a pretty strict check. The user has to get the exact item name, including the correct case. But I can't think of another checking method that is strict enough, while also being forgiving. Feel free to not change this.

**Comment by rosswf** [comment]
Thank you for taking the time to review. I will definitely have a look at these issues.

Is there a way I can run the test locally prior to updating the pull request?

File: osrsbox/items_api_examples/guess_the_item_game.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+and asks the user to guess the name of the item
+"""
+
+
+from osrsbox import items_api
+import random
+
+if __name__ == "__main__":
+    all_db_items = items_api.load()
+    item_found = False
```
[the remark this answers is not in the document]
Comment (rosswf) [reply osrsbox/items_api_examples/guess_the_item_game.py]: The variable `item_found` is used to break the `while item_found is False` loop. Is there a cleaner way to do this? The purpose of this loop is to ensure that an item with no examine text isn't chosen.

[the remark this answers is not in the document]
Comment (osrsbox) [reply osrsbox/items_api_examples/guess_the_item_game.py]: Ah! Yes, you are correct. That implementation is completely fine. I must have not had enough coffee when first looking at it!

**Comment by osrsbox** [comment]
If you want to check the code style/conventions before submitting a PR:

`pip install flake8`

`python -m flake8` (in the root project folder named `osrsbox-db`)

**Pull request merged**